css/style.css

```diff
@@ -0,0 +1,17 @@
+#image {
+    display: block;
+    max-width: 100%;
+    height: auto;
+    margin-left: auto;
+    margin-right: auto;
+
+}
+
+#main{
+    text-align: center;
+}
+
+#img-div a{
```

Comment:
I think it's better to avoid overusing id selectors, instead it's better to use class selectors. This makes your code more reusable. 💯